code/game/objects/items/trader.dm

```diff
@@ -62,6 +62,7 @@
 /obj/structure/closet/secure_closet/wonderful/New()
 	..()
 	var/random_clothes = clothing.Copy()
+	random_clothes = random_clothes - typesof(/obj/item/clothing/mask/stone) - typesof(/obj/item/clothing/mask/morphing) - typesof(/obj/item/clothing/accessory/holomap_chip) - typesof(/obj/item/clothing/suit/space/time) - typesof(/obj/item/clothing/head/helmet/space/time) - typesof(/obj/item/clothing/gloves/warping_claws)
```

Comment:
Triple check this works

Reply:
I spawned 400 closets and sdql turned up no stone masks or warping claws in the closets (but there were some of those in nullspace presumably due to the Copy()).

Reply:
Odd. Why would it be spawing them in the first place. Thank you for testing

Reply:
(No clue how to approve changes via mobile)